### ISSUE TITLE
Allow easy direct checking of public keys if you have them

### ISSUE THREAD

**Issue by joerocklin** [trufflesecurity/driftwood#4]
Security people get a little twitchy with things touching the private keys, it would be great if this supported reading the public keys if someone already has them.

**Comment by dustin-decker** [comment]
You're right, and we'll add this soon. In the meantime the code base is small and auditable, and the builds are automated.
